packages/tools/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "@manypkg/tools",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/manypkg-tools.cjs.js",
+  "dependencies": {
+    "@babel/runtime": "^7.20.6",
+    "fs-extra": "^8.1.0",
+    "globby": "^11.0.0",
+    "jju": "^1.4.0"
```

`PnpmTool` needs this:
```suggestion
    "jju": "^1.4.0",
    "read-yaml-file": "^1.1.0"
```